blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -64,8 +74,57 @@ const HeaderAccountAction = ({ customFields }) => {
 
   // What do we want to happen if there is an error?
   return (
-    <div className="xpmedia-subs-header">
-      <a href={loginURL} className="xpmedia-subs-header--button">Sign In</a>
+    <div className="xpmedia-subs-header--logged-out-header">
+      <div className="xpmedia-subs-header--desktop-logged-out-header">
+        <Button
```

Comment:
Each button should only render if they have a value set for the `href`

Reply:
updated. as well for the dropdown links. thanks. will write tests as well